core-web/src/main/java/org/silverpeas/core/web/treemenu/process/TreeBuilder.java

```diff
@@ -96,16 +94,20 @@ private static MenuItem buildOtherLevel(TreeFilter filter, MenuItem father, Stri
     // the first level of theme
     if (father.getType() == NodeType.COMPONENT && filter.acceptNodeType(NodeType.THEME)) {
       NodePK nodePK = new NodePK("0", father.getKey());
-      Collection<NodeDetail> nodeDetails = getNodeBm().getChildrenDetails(nodePK);
+      List<NodeDetail> nodeDetails = new ArrayList<>(getNodeBm().getChildrenDetails(nodePK));
+
+      if (TreeHandler.useOrder)
+        nodeDetails.sort(Comparator.comparing(NodeDetail::getOrder));
+
```

Comment:
A Java rule: always the body of an if or else-statements is put in brackets, even if there is only a single statement of code

Reply:
Done